### PR TITLE
Input image

### PR DESCRIPTION
**Pull request by shlok30**
Hi Team,

Can you please review my image and input components

Input - https://water-style-ui.netlify.app/components/input.html
Image - https://water-style-ui.netlify.app/components/image.html

My overall site is not responsive yet as its something that I will handle after the submission.

Thanks in advance

### CODE REVIEW

**Review by haarsh24 — APPROVED** [review]
Everything looks good, you can merge.

A small suggestion- try to add more description in the commit message (instead of a bug fix, add what bug you have fixed, it will help you in the future).

File: Components/input.html [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+        </div>
+
+        <iframe
+          src="https://carbon.now.sh/embed?bg=rgba%2859%2C130%2C246%2C1%29&t=seti&wt=none&l=auto&width=503&ds=true&dsyoff=20px&dsblur=68px&wc=false&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=10.5px&lh=142%25&si=false&es=2x&wm=false&code=%253Clabel%2520for%253D%2522demo%2522%2520class%253D%2522input-label%2522%253ELabel%253C%252Flabel%253E%250A%253Cinput%2520type%2520%253D%2520%2522text%2522%2520id%2520%253D%2520%2522demo%2522%2520class%2520%253D%2520%2522input-field%2522%2520placeholder%253D%2522Please%2520Enter%2520Text%2520Here%2522%253E"
+          style="
+            width: 503px;
+            height: 71px;
+            border: 0;
+            transform: scale(1);
+            overflow: hidden;
```
Comment (akash-1318) [inline Components/input.html:59]: don't use inline styles for iframes just add a class in util i think .

**Review by akash-1318 — COMMENTED** [review]
Looks good to merger just one suggestion mentioned in comment.

File: Components/input.html (hunk 1 above)
Comment (shlok30) [reply Components/input.html:59]: Hey, this i-frame was directly copied from carbon.now.sh, but yeah it makes sense to put it in a class maybe

**Pull request merged**